src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -167,7 +169,12 @@ public function export(array $table): array
         ];
         // output state
         if (!empty($result['lastFetchedRow'])) {
-            $output['state']['lastFetchedRow'] = $result['lastFetchedRow'];
+            if ($this->canFetchMaxIncrementalValueSeparately($isAdvancedQuery)) {
+                $maxValue = $this->getMaxOfIncrementalFetchingColumn($table['table']);
```

Comment:
This should be done before the export.  There is a chance that some rows after export and until this point may be missing.

Reply:
see https://github.com/keboola/db-extractor-mssql/blob/master/src/Keboola/DbExtractor/Extractor/MSSQL.php#L217

Reply:
fixed... thanks 👍